cg/meta/workflow/analysis.py

```diff
@@ -100,6 +100,13 @@ def get_slurm_qos_for_case(self, case_id: str) -> str:
         priority: int = self.get_priority_for_case(case_id=case_id)
         return PRIORITY_TO_SLURM_QOS[priority]
 
+    def get_workflow_manager(self) -> str:
+        """Get task manager for a given pipeline."""
+        if hasattr(self, "workflow_manager"):
+            return self.workflow_manager
+        else:
+            return WorkflowManager.Slurm.value
```

Comment:
```suggestion
        return WorkflowManager.Slurm.value
```
you do not need the else since you return in the if condition